template-contributing-guide.md

```diff
@@ -0,0 +1,341 @@
+- **Pull request reviewers** - This group of experienced templateers review and approve pull requests submitted to the templates repository.
+
+In addition to these roles, these resources are available to help you as you contribute templates to our project:
+
+- **Working groups** - Our project is organized into several working groups that meet on a regular basis to work on the project’s key initiatives. To effectively contribute to a project, consider joining one of these working groups.
+- **Templates issue list** - All the templates that are actively being worked on or which could be worked on have a corresponding issue in the templates repository. Use the issue list or the kanban board to find a template to work on and track your template’s progress as it moves throughout the template writing phases.
+
+These roles and resources (including the names of the individuals currently serving in these roles) are explained in the [Template Roles and Resources](template-roles-and-resources.md) guide.
+
+
+### Required deliverables
+
+Currently, for a template set to be considered complete, each template set should have these documentation deliverables:
+
+<table>
+  <tr>
+    <th>Deliverable</th>
+    <th>Description and purpose</th>
+    <th>Required?</th>
+  </tr>
+  <tr>
+    <td>Raw template</td>
+    <td>The raw template is the basic template for the type of document you are creating. It provides a rough outline of the suggested content and a few embedded writing tips for how to fill in the different sections of the template.</td>
+    <td>Yes, required</td>
+  </tr>
+  <tr>
+    <td>Template guide</td>
+    <td>This guide provides a deeper explanation of how to fill in the raw template. It provides a lightweight introduction to the purpose of this documentation and explains how to fill in each section of the document. Additional content may be included as needed.</td>
+    <td>Yes, required</td>
+  </tr>
+  <tr>
+    <td>Before you start guide</td>
+    <td>This optional guide is designed to help template users understand some of the key research, brainstorming, or pre-writing steps that they might need to do before adding this type of document to a guide. If this documentation type has certain key decisions that need to be made with stakeholders or if it has a dependency on some other type of document or process, that should be mentioned in this guide.</td>
+    <td>No, optional</td>
+  </tr>
+  <tr>
+    <td>Template example</td>
+    <td>The Good Docs Project is in the process of designing a fake documentation project that can provide examples of our templates in action. Until this project is ready, it is optional to provide an example of the template. However, if you want to create a fake example as part of your template set, you may do so.</td>
+    <td>No, optional</td>

```

Comment:
I personally think this should be required. It's literally my best resource when I'm looking at templates, because of the way my brain works. It should be a trivial ask.

Reply:
Unfortunately, it has to be optional since our example project is not available yet.

Reply:
I've noted this comment in the feedback section of the RFC for now.